mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -865,8 +866,9 @@ is set to true. Defaults to ZONAL.`,
 				Description: `The configuration for replication.`,
 			},
 			"server_ca_cert": {
-				Type:     schema.TypeList,
-				Computed: true,
+				Type:       schema.TypeList,
+				Computed:   true,
+				Sensitive:  true,
```

Comment:
ditto here

Reply:
Thanks, Nick. Unfortunately this is a limitation of TF for these nested messages. Please see [this issue](https://github.com/hashicorp/terraform-plugin-sdk/issues/201). So I have to mark the whole message as sensitive.

I've tested the version which only marks the sensitive fields in the nested message. TF failed to hide them.